frontend/src/router/router.ts

```diff
@@ -0,0 +1,23 @@
+import { createRouter, createWebHistory } from "vue-router";
+
+import LoginPage from '@/views/LoginPage.vue';
+
+const router = createRouter({
+  history: createWebHistory(),
+  routes: [
+    {
+      path: "/login",
+      component: LoginPage

```

Comment:
[FYI][can be ignored] it is advised to use dynamic imports for your routes to minimize load times.
https://router.vuejs.org/guide/advanced/lazy-loading.html
Please note that without dynamic imports all your components will be loaded on any page visit.

Reply:
Done